### DISCUSSION TITLE
Weblate fails to start after switching disks

### DISCUSSION THREAD

**Discussion by chimosky** [WeblateOrg/weblate#10625] [Q&A] [answered]
I'd recently switched the disk weblate used to run on to a bigger one as the old one was almost filled and I'd copied the contents of the old disk to the new one using `dd`, there was no db backup and after I'd copied pointed weblate to the new disk - running a self hosted weblate instance - weblate fails to start with this error.

```bash 
Starting...
System check identified no issues (1 silenced).
Postgres 160000 is up
Starting database migration...
Operations to perform:
  Apply all migrations: accounts, addons, admin, auth, authtoken, checks, configuration, contenttypes, django_celery_beat, fonts, gitexport, glossary, lang, memory, metrics, screenshots, sessions, social_django, trans, utils, vcs, weblate_auth, wladmin
Running migrations:
  No migrations to apply.
[2023-12-14 12:49:29,411: INFO/69] Acquired Lock('lock:lock:gitlock').
CommandError: Multiple users matched given parameters!
```

**Comment by nijel** [comment, marked as answer]
This is unrelated to disk switch.

Your admin settings (https://docs.weblate.org/en/latest/admin/install/docker.html#envvar-WEBLATE_ADMIN_NAME) differs from the initial setup, so there are more users matching. Change WEBLATE_ADMIN_EMAIL to the actual e-mail of the `admin` user.

**Reply by nijel** [reply]
https://github.com/WeblateOrg/weblate/pull/10638 should improve the error message.

**Reply by chimosky** [reply]
> This is unrelated to disk switch.
> 
> Your admin settings (https://docs.weblate.org/en/latest/admin/install/docker.html#envvar-WEBLATE_ADMIN_NAME) differs from the initial setup, so there are more users matching. Change WEBLATE_ADMIN_EMAIL to the actual e-mail of the `admin` user.

Yeah, I'd seen your response in another thread and changed `WEBLATE_ADMIN_EMAIL` to the last known admin email and that didn't work but not setting it at all got weblate to start and trying to change it in the container returned this error;

```bash
CommandError: Multiple users matched given parameters!
```

Thanks for the help.
